### PR TITLE
Added a Random Password Generator

### PR DESCRIPTION
**Pull request by gowiredu95**
A password generator for new users and users who forgot their password. And for anything else.

### CODE REVIEW

File: src/java/Controller/PasswordGenerator.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ */
+
+
+/**
+ *
+ * @author syntel
+ */
+public class PasswordGenerator {
+
+    /*
```
Comment (JSauceda10) [inline src/java/Controller/PasswordGenerator.java]: The main fucntion to be deleted per Chris's request.

**Review by apj11c — APPROVED** [review]

File: src/java/Controller/PasswordGeneratorEngine.java [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+        if (useUpper) {
+            charCategories.add(UPPER);
+        }
+        if (useDigits) {
+            charCategories.add(DIGITS);
+        }
+        if (usePunctuation) {
+            charCategories.add(PUNCTUATION);
+        }
+
```
Comment (JSauceda10) [inline src/java/Controller/PasswordGeneratorEngine.java:118]: Looks good

**Review by JSauceda10 — COMMENTED** [review]
Looks good

**Review by ChrisSMendoza — APPROVED** [review]
Solid.

**Pull request merged**